cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -438,6 +463,61 @@ message InstalledPackageSummary {
   InstalledPackageStatus status = 11;
 }
 
+// InstalledPackageDetail
+//
+// An InstalledPackageDetail includes details about the installed package that are
+// typically useful when presenting a single installed package.
+message InstalledPackageDetail {
+  // InstalledPackageReference
+  //
+  // A reference uniquely identifying the package.
```

Comment:
i guess this is the better place to comment..
should we say "installed package" instead of only "package"?